src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/ApiTests/InstanceApiTest.cs

```diff
@@ -26,6 +27,8 @@ public class InstanceApiTest : IClassFixture<CustomWebApplicationFactory<Altinn.
         public InstanceApiTest(CustomWebApplicationFactory<Altinn.App.Startup> factory)
         {
             _factory = factory;
+
+            EventsMockSI.Requests.Clear();
```

Comment:
wondering if it makes more sence to clear this in the begining of the test rather than the constructor? If we we're to add more tests in the future for events that might be a more fool proof practice?

Reply:
The constructor is run once for every test. The XUnit test runner will create one instance of the test class for every test. This is a major difference between XUnit and other frameworks we have used.

Reply:
But, I do run into issues here for completely different reasons. I made the Requests static so it can be accessed when the test has completed, but now two tests can ruin it for each other. I don't know of any easy fix for this so I'll just remove the Asserts.